cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,80 @@
+describe('EngageSphere API - GET /customers', () => {
+  const baseUrl = 'http://localhost:3001/customers';
+
+  it('should retrieve customers with default query parameters', () => {
+    cy.request(baseUrl).then((response) => {
```

Comment:
What about using object destructuring here so we don't need to repeat `response.` all over the place.